core/domain/exp_jobs_one_off.py

```diff
@@ -202,7 +202,6 @@ def entity_classes_to_map_over(cls):
     def map(item):
         if item.deleted:
             return
-
```

Comment:
Probably don't change this file just for one blank line.

Reply:
Done!